packages/config/config/devices/0x0312/mp20z.json

```diff
@@ -14,10 +14,118 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"paramInformation": {
+		"1": {
+			"label": "Status LED Configuration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED on when load on, LED off when either load off",
+					"value": 0
+				},
+				{
+					"label": "LED on when load off, LED off when either load on",
+					"value": 1
+				},
+				{
+					"label": "LED always off",
+					"value": 2
+				}
+			]
+		},
+		"2": {
+			"label": "Left Outlet Auto-Off Timer",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"3": {
+			"label": "Right Outlet Auto-Off Timer",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"4": {
+			"label": "Left Outlet Auto-On Timer",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"5": {
+			"label": "Right Outlet Auto-on Timer",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"6": {
+			"label": "State After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Output Off",
+					"value": 0
+				},
+				{
+					"label": "Output On",
+					"value": 1
+				},
+				{
+					"label": "Restore Outlets to Previous State",
+					"value": 2
+				}
+			]
+		}
+	},
```

Comment:
We have [several templates](https://github.com/zwave-js/node-zwave-js/blob/086ac3e67a36f74e4157c46c3511529d22700520/packages/config/config/devices/templates/master_template.json#L256) for these, please choose the matching one and [use it](https://zwave-js.github.io/node-zwave-js/#/config-files/using-templates?id=using-templates).

Reply:
Well... that was meant to go on the "State after power loss" parameter.

Reply:
Done!